CMakeLists.txt

```diff
@@ -37,14 +37,14 @@ add_custom_command(
 
 add_custom_target(
     test
-    ${CMAKE_COMMAND} -E env PACKCC=${PACKCC} BUILDDIR=${CMAKE_CURRENT_BINARY_DIR} tests/run.sh -T
```

Comment:
I'd like to see the timing when I run this manually on my machine. Ubuntu has bats 1.10 (since Ubuntu 23.10), which supports `-T`, so we can use Ubuntu 24.4 image and keep this. 

Alternatively, the last step in the workflow can be running the `tests/run.sh` directly, which can be done without the flag.